app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -646,6 +647,17 @@ export default Controller.extend({
       return;
     }
 
+    const topic = composer.topic;
+
+    if (topic.slow_mode_seconds && topic.user_last_posted_at) {
```

Comment:
Since we already know that the post can be submitted, I chose to show the error earlier. I'm fine with displaying the error from the backend, do you prefer that instead?